WooCommerce/Classes/POS/Utils/StockFormatter.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import struct Yosemite.POSSimpleProduct
+
+struct StockFormatter {
+    static func stockStatusLabel(for product: POSSimpleProduct) -> String {
+        switch product.manageStock {
+        case false:
+            return product.productStockStatus.description
+        case true:
+            guard let stock = product.stockQuantity else {
+                return product.productStockStatus.description
+            }
+            if stock <= 0 {
+                return Localization.outOfStock
```

Comment:
nit: could this reuse the out-of-stock string as in `ProductStockStatus.outOfStock.description`?

Reply:
Interesting, I see `ProductStockStatus` included in the Yosemite model and the enum description getter is public. I thought it'd be possible to use `ProductStockStatus` from Yosemite.

https://github.com/woocommerce/woocommerce-ios/blob/a9eae7b93fb1f1d4a8b2c62ecf88975291b1db21/Yosemite/Yosemite/Model/Model.swift#L109

Reply:
You are very right, sorry for that. When I tried I must have miss-read the access error 🤔  I'll make the change when working through variations next.